style/git/.gitmessage

```diff
@@ -0,0 +1,40 @@
+
+# <type>(<scope>): <subject>
+#
+# <body>
+#
+# <footer>
+#
+# ** TYPES **
+# feat (new feature)
+# fix (bug fix)
+# docs (changes to documentation)
+# style (formatting, missing semi colons, etc; no code change)
+# refactor (refactoring production code)
+# test (adding missing tests, refactoring tests; no production code change)
+# chore (updating grunt tasks etc; no production code change)
+#
+# ** FOOTERS **
+# References #1, #4, and #2.
+# Fix #1. note this marks the item as accepted in Sprintly
```

Comment:
We don't really use Sprintly anymore, change this to Github? Bonus points: include link to page explaining how `closes #2` works?